include/deal.II/base/mpi_remote_point_evaluation.h

```diff
@@ -111,17 +111,52 @@ namespace Utilities
              const Mapping<dim, spacedim> &mapping);
 
       /**
-       * Data of points positioned in a cell.
+       * Helper class to store and to access data of points positioned in
+       * processed cells.
        */
-      struct CellData
+      class CellData
```

Comment:
With the new API, one can write readable code like: https://github.com/dealii/dealii/blob/5fcce7b0d5949a3c38e528b07d5a790878d6e4b2/examples/step-87/step-87.cc#L388-L395